### ISSUE TITLE
enhance overall ui

### ISSUE THREAD

**Issue by itsgeekhokage** [Sujal-2820/Hacktoberfest-Web#12]
@Sujal-12345 please assign me this

**Comment by Sujal-2820** [comment]
Hey @Shivam36998,
It's good that you are showing interest in contributing to the project 🙂

Can you describe what all changes are you willing to implement in order to enhance the UI..

**Comment by Shivam36998** [comment]
I think there is far scope of improvement here, 

- navbar can be made better and fixed
- we can maintain images as cards, giving them proper shape and size and arranging them in a well manner

**Comment by Shivam36998** [comment]
@Sujal-2820 please assign me this, if you have anything that you want on it, please say it

**Comment by akshitjain2004** [comment]
Hey @Sujal-2820 . I created an image slider for this project. Which is automatic and manual. Assign me this issue. I'll create a PR and I'll make a few more changes which I'll explain in the PR
![image](https://github.com/Sujal-2820/Hacktoberfest-Web/assets/94056853/9dbc3911-47ff-4287-b109-d8b984c07b00)

**Comment by Sujal-2820** [comment]
@akshitjain2004 please make a new issue for the changes you want to make, I'll then assign it to you. All the Best 👍

**Comment by akshitjain2004** [comment]
@Sujal-2820 Yes. I have created a new issue and assign it to me. I'll do it.

**Comment by Shivam36998** [comment]
@Sujal-2820 i made a pr request
